src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -2108,27 +2111,6 @@ internal override bool IsReadOnly
             }
         }
 
-        //PROTOTYPE(span): Span and ReadonlySpan should have spanLike marker.
-        //                 For now assume that any "System.Span" and "System.ReadOnlySpan" structs 
-        //                 are span-like
-        private bool IsWellknownSpans()
-        {
-            var originalDef = this.OriginalDefinition;
-
-            if (originalDef.Name != "Span" && originalDef.Name != "ReadonlySpan")
-            {
-                return false;
-            }
-
-            var ns = originalDef.ContainingSymbol as NamespaceSymbol;
-            if (ns?.Name != "System")
-            {
-                return false;
-            }
-
-            return ns.IsGlobalNamespace;
```

Comment:
Something doesn't look right to me here. If `return ns.IsGlobalNamespace` was the existing check, it should never have returned true, thus tests should not be passing for any positive type.
Am I missing something here?